build/installers/liferay-workspace/liferay-workspace.xml

```diff
@@ -359,6 +359,17 @@
         <deleteFile>
             <path>${installdir}</path>
         </deleteFile>
+        <deleteFile>
+            <path>C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Liferay Workspace</path>
```

Comment:
Can we not just not install this to begin with?  instead of deleteing it?

What if they don't have a C:\ drive?